Makefile

```diff
@@ -6,4 +6,5 @@ image:
 	docker build --rm -t $(IMAGE_NAME) .
 
 test: image
-	docker run --rm $(IMAGE_NAME) sh -c "cd /usr/src/app && bundle exec rake"
+	docker run --rm $(IMAGE_NAME) sh -c "cd /usr/src/app && \
+		CODECLIMATE_REPO_TOKEN=ca5120f90202a99c43c0914d301a94e5908136d146ecc0d7fe7fde1c3f9d325e bundle exec rake"
```

Comment:
I think we generally hardcode the token these days, because relying on it to be passed in via the environment and then setting it in the environment on the CI service means that cross-origin contributions (which are denied access to environment variables on the CI service) don't get test coverage. Taking the `--workdir` suggestion... let me know if you'd like me to change the token bit of it as well, with that context